atomate/vasp/firetasks/glue_tasks.py

```diff
@@ -202,6 +203,52 @@ def run_task(self, fw_spec):
         return FWAction(stored_data=stored_data)
 
 
+@explicit_serialize
+class GetInterpolatedPOSCAR(FiretaskBase):
+    """
+    Grabs CONTCARS from two previous calculations to create interpolated
+    structure.
+
+    Required params:
```

Comment:
Many of these parameters don't seem to be required, i.e., the code will run perfectly well without them being set. It's better if you can decide what should really be required and what should really be optional. If it is a required parameter, i.e., you really think the user should be setting it and the code shouldn't be setting an intelligent default, then the code should FAIL if it is not set.

Reply:
I've altered the code so now many of these parameters are required.